airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -734,6 +756,7 @@ private ConnectionRead createConnection(String name,
             .destinationId(destinationId)
             .syncCatalog(catalog)
             .schedule(schedule)
+            .operationIds(operationId == null ? Collections.emptyList() : List.of(operationId))
```

Comment:
Wouldn't it be cleaner to get `List<UUID> operationIds` instead of `UUID operationId` as additional argument to `createConnection`?
That way you can pass an empty list of operations instead of null here.

Later, we'll be able to create multiple operations for the test too (if we ever need it)